automation/services/watchdog/check_libp2p/check_libp2p.go

```diff
@@ -31,11 +31,13 @@ func main() {
 	for i, addr := range addrs {
 		maddr, err := ma.NewMultiaddr(addr)
 		if err != nil {
+			fmt.Fprintln(os.Stderr,err)
```

Comment:
Been talking to @noot about this -- she thinks the failures may have something to do with the chain id not being set in this program.